great_expectations/expectations/core/expect_query_results_to_match_source.py

```diff
@@ -136,6 +154,74 @@ def schema_extra(
                 }
             )
 
+    @classmethod
+    def _get_query_rendered_content(
+        cls,
+        renderer_configuration: RendererConfiguration,
+        query_type: Literal["source", "target"],
+        add_param_args: AddParamArgs,
+        template_str: Optional[str] = None,
+    ) -> RenderedAtomicContent:
+        for name, param_type in add_param_args:
+            renderer_configuration.add_param(name=name, param_type=param_type)
+
+        renderer_configuration.template_str = template_str
+
+        renderer_configuration.code_block = CodeBlock(
+            code_template_str=f"${query_type}_query",
+            language=CodeBlockLanguage.SQL,
+        )
+
+        value_obj = renderedAtomicValueSchema.load(
```

Comment:
I've seen this pattern elsewhere. When we have the pieces in memory, why use the schema at all when we can just pass it into `RenderedAtomicValue` directly?